.gitignore

```diff
@@ -32,6 +32,9 @@ atlassian-ide-plugin.xml
 # code review
 codereview.rc
 
+# evergreen
+expansion.yml
```

Comment:
Added so that expansion.yml file created in pre tasks in Evergreen doesn't make the working directory dirty